frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/nodes-selection-list.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Text, pluralize } from '@patternfly/react-core';
+import * as classNames from 'classnames';
+import { sortable, OnSelect } from '@patternfly/react-table';
+import { Table } from '@console/internal/components/factory';
+import {
+  ResourceLink,
+  humanizeBinaryBytes,
+  convertToBaseValue,
+} from '@console/internal/components/utils';
+import { NodeKind } from '@console/internal/module/k8s';
+import { getUID, getName, getNodeCPUCapacity, getNodeAllocatableMemory } from '@console/shared';
+import { NodeTableRow, RowUIDMap } from './types';
+import { getSelectedNodeUIDs } from './utils';
+import './node-selection-list.scss';
+
+const tableColumnClasses = [
+  classNames('pf-u-w-30-on-sm'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm', 'pf-u-w-10-on-sm'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm', 'pf-u-w-20-on-sm'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm', 'pf-u-w-20-on-sm'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm', 'pf-u-w-20-on-sm'),
+];
+
+const setTableHeader = () => {
+  return [
+    {
+      title: 'Name',
+      sortField: 'metadata.name',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'CPU',
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'Memory',
+      props: { className: tableColumnClasses[2] },
+    },
+    {
+      title: 'Location',
+      props: { className: tableColumnClasses[3] },
+    },
+    {
+      title: 'Taints',
+      props: { className: tableColumnClasses[3] },
+    },
```

Comment:
Add sorts for all these fields as well.

Reply:
It as per UXD

Reply:
@yuvalgalanti why are we only allowing users to sort on the basis of Name? IMO we should sort on CPU and Memory too.